x/scheduler/types/keeper.go

```diff
@@ -1,18 +1,19 @@
 package types
 
 import (
-	"github.com/cometbft/cometbft/libs/log"
+	"context"
+	"cosmossdk.io/log"
 	sdk "github.com/cosmos/cosmos-sdk/types"
-	authtypes "github.com/cosmos/cosmos-sdk/x/auth/types"
+	"github.com/cosmos/cosmos-sdk/x/auth/types"
 )
 
 //go:generate mockery --name=Keeper
 type Keeper interface {
 	AddNewJob(ctx sdk.Context, job *Job) error
-	GetAccount(ctx sdk.Context, addr sdk.AccAddress) authtypes.AccountI
+	GetAccount(ctx context.Context, addr sdk.AccAddress) types.AccountI
 	GetJob(ctx sdk.Context, jobID string) (*Job, error)
-	Logger(ctx sdk.Context) log.Logger
-	PreJobExecution(ctx sdk.Context, job *Job) error
+	Logger(ctx context.Context) log.Logger
+	PreJobExecution(ctx context.Context, job *Job) error
```

Comment:
Some of the keeper functions are using `sdk.Context`, now some others are using `context.Context`. Where's the difference coming from and can we align on just using one of them?

Reply:
We replaced the structs defined within paloma modules and types to `context.Context` we did not replace the ones which were defined by cosmos-sdk struct. [Here](https://github.com/cosmos/cosmos-sdk/blob/7d5c2dbae68530bc9070983ce0b2c8a1efe23412/x/group/keeper/keeper.go#L238) is the sample keeper from cosmos-sdk which still uses sdk.context

Reply:
What's the idea behind this? Why not use `sdk.Context` for custom handlers as well?

Reply:
Sure, I just checked there were few ununsed types which were using `sdk.context` now updated to `context.Context`